peps/pep-0770.rst

```diff
@@ -380,9 +383,68 @@ For projects that do contain other software components, documentation will be
 added to the Python Packaging User Guide for how to specify and maintain
 SBOM documents for Python packages in source code.
 
+There are two "camps" of projects that contain other software, those from
+a "packaging ecosystem" (PyPI, Linux distros, Rust, NPM, etc) and those from
```

Comment:
Seems odd that we say NPM/PyPI for JS and Python, but Rust instead of https://crates.io
```suggestion
a "packaging ecosystem" (PyPI, Linux distros, Crates.io, NPM, etc) and those from
```